app/src/main/java/ch/epfl/sdp/game/data/Participation.kt

```diff
@@ -0,0 +1,11 @@
+package ch.epfl.sdp.game.data
+
+import ch.epfl.sdp.lobby.PlayerParametersFragment
+import ch.epfl.sdp.user.User
+
+class Participation (
+        var user: User = User(),
+        var ready: Boolean = false,
+        var tag: String = "",
+        var faction: PlayerParametersFragment.Faction = PlayerParametersFragment.Faction.PREY
+)
```

Comment:
Same criticism as for the Game class:
Why use default values ? To me it allows the user of this class to forget a field.
If a default Participation object is really needed sometimes, a factory returning a DefaultEmptyParticipation could do the trick

Reply:
Again, case class